editor/resources/templates/template.glsl

```diff
@@ -0,0 +1,5 @@
+// Put functions in this file to use them in shaders
+// To get access to the functions, you need to put:
+// #include "/my-folder/my-file.glsl"
+// in any script using the functions.
+// Please consult the manual on how to use this feature!
```

Comment:
We could include a link to the manual here but the entry doesn't exist yet since this hasn't been merged yet..